.travis.yml

```diff
@@ -37,5 +37,7 @@ install:
     - obvci_install_conda_build_tools.py
 
 script:
-    - obvci_conda_build_dir.py ./ SciTools --channel main --build-condition "numpy >=1.8" "python >=2.7,<3|>=3.4,<3.5"
```

Comment:
This breakdown shouldn't be necessary (I appreciate it is probably just an attempt to separate the concerns so that it can be tested)